CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
+## 0.3.0 - TBD
```

Comment:
Since there is no BC break, it can stay in the 0.2 series. I'll make that change on merge.